__tests__/app.test.js

```diff
@@ -56,3 +56,47 @@ describe("/api/reviews", () => {
       });
   });
 });
+
+describe('"/api/reviews/:reviews_id', () => {
+  describe("GET", () => {
+    test("200: returns an object containing information on requested review (with comment count)", () => {
+      return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review).toEqual({
+            review_id: 3,
+            title: "Ultimate Werewolf",
+            designer: "Akihisa Okui",
+            owner: "bainesface",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "We couldn't find the werewolf!",
+            category: "social deduction",
+            created_at: "2021-01-18T10:01:41.251Z",
+            votes: 5,
+            comment_count: "3",
+          });
+        });
+    });
+  });
+
+  test("404: returns an error message when passed correct data type but a review_id that does not exist", () => {
+    return request(app)
+      .get("/api/reviews/99999")
+      .expect(404)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body.message).toBe("Review ID does not exist");

```

Comment:
Only thing I would say is make your own error message keys 'msg' instead of 'message' so we can spot what is our error and what is a more 'official' error from PSQL or node etc.